en/camera/index.md

```diff
@@ -1 +1,99 @@
 # Cameras
+
+Cameras are important for many [payload use cases](../payloads/use_cases.md), including mapping and surveying, surveillance, search & rescue, crop health and pest detection, and so on.
+They are commonly mounted on a [gimbal](../advanced/gimbal_control.md) that can provide camera stabilisation, point tracking, and movement independent of the hosting vehicle.
+
+## Camera Types
+
+PX4 integrates with two types of cameras (follow links for setup/configuration information):
+
+- [MAVLink cameras](../camera/mavlink_camera.md) that are controlled using the [MAVLink camera protocol](https://mavlink.io/en/services/camera.html).
+
+  These provide full access to any feature of the camera that has been exposed via the protocol, such as image and video capture, video streaming to a ground station, control over zoom and focus, selecting between infrared and visible light feeds, and so on.
+
+  Usually a camera manager running on a companion computer is used to interface between MAVLink and a camera's native protocol.
+
+- [Cameras Attached to Flight Controller Outputs](../camera/fc_connected_camera.md)
+
+  Photo capture can be triggered from a flight controller pin, and in some cases an additional pin can toggle camera modes.
+
+Both approaches allow image capture in missions and when commanded by a ground station, and are suitable for mapping and survey applications.
+MAVlink cameras support many more functions, such as streaming and video capture, and can be used for a broader range of applications.
+
+## Overview
+
+### Mission Camera Commands
+
+PX4 _allows_ the following camera commands in missions (i.e. does not reject the mission if they are included):
```

Comment:
Note to me. If I'm right that Mavlink camera doesnt need the camera driver, then this should be modified. Part of that would be to explain that PX4 just emits all camera commands, both old and new, and that they are different and used by different cameras. Lets see.